etcdctl/ctlv3/ctl.go

```diff
@@ -42,10 +45,14 @@ var (
 		Use:        cliName,
 		Short:      cliDescription,
 		SuggestFor: []string{"etcdctl"},
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			HandleConfigFile(cmd)
+		},
 	}
 )
 
 func init() {
+	rootCmd.PersistentFlags().StringVarP(&globalFlags.ConfigFile, "config-file", "c", "", "Path to the client configuration file. Note that other command line flags and environment variables take precedence.")
```

Comment:
`Path to the configuration files for etcdctl. Note that explicitly provided command-line flags and environment variables take precedence.`